project_resource_calendar/i18n/fr.po

```diff
@@ -238,25 +248,40 @@ msgid "Description"
 msgstr "Description"
 
 #. module: project_resource_calendar
-#: model:ir.model.fields,field_description:project_resource_calendar.field_doublebooking_validation_wiz_display_name
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_calendar_wizard.js:111
+#, python-format
+msgid "Discard"
+msgstr "Discard"
```

Comment:
why not translated

Reply:
I wasn't originally in the POT file I downloaded from Odoo to add new translations.